app/controllers/public/index.js

```diff
@@ -10,6 +11,19 @@ export default Controller.extend({
   { name: 'Speaker 4', organisation: 'Org 4', isFeatured: true, socialLinks: [{ name: 'twitter', url: '#' }], short_biography: 'Full-stack developer with experience in PHP, Python &amp; Ember.JS. Plays around with Kubernetes and Containers. Avid open source contributor.' },
   { name: 'Speaker 5', organisation: 'Org 5', isFeatured: false, socialLinks: [{ name: 'home', url: '#' }],  short_biography: 'Full-stack developer with experience in PHP, Python &amp; Ember.JS. Plays around with Kubernetes and Containers. Avid open source contributor.' }],
 
+  sponsors: [{ name: 'Sponsor 2', url: '#', logoUrl: 'http://placehold.it/150x60', level: 2, type: 'Gold Sponsor', description: 'Blah Blah Blah' },
+    { name: 'Sponsor 1', url: '#', logoUrl: 'http://placehold.it/150x60', level: 1, type: 'Gold Sponsor', description: 'Blah Blah Blah' },
+    { name: 'Sponsor 3', url: '#', logoUrl: 'http://placehold.it/150x60', level: 1, type: 'Silver Sponsor', description: 'Blah Blah Blah' }
+  ],
+
+  sponsorsByCategory: computed(('sponsors'), function() {
```

Comment:
And also, since we want the computed property to observe element changes in the array, 
it needs to be 
```js
sponsorsByCategory: computed('sponsors.[]', function() {
```

If you use just `sponsors`, the property will recompute only if for example `sponsors` changes from `null` to an array or back. The property will **not** recompute if for example, an item is added to the `sponsors` array.

But we want it to be recomputed if the array elements are added/removed. So we specify it as 
```js
sponsorsByCategory: computed('sponsors.[]', function() {
```